CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+<a name="2.1.0"></a>
+# [2.1.0](https://github.com/zalando/dress-code/compare/2.0.0...v2.1.0) (2017-08-22)
+
+
+### Bug Fixes
+
+* **style:** Standardize elements height to 2.4rem ([6a54b3e](https://github.com/zalando/dress-code/commit/6a54b3e)), closes [#323](https://github.com/zalando/dress-code/issues/323)
+* **table:** fix possible overlapping of sorters ([46aa548](https://github.com/zalando/dress-code/commit/46aa548))
+
+
+### Code Refactoring
+
+* **breakpoints:** use same breakpoints for media queries and grid classes ([#360](https://github.com/zalando/dress-code/issues/360)) ([82eeb42](https://github.com/zalando/dress-code/commit/82eeb42)), closes [#239](https://github.com/zalando/dress-code/issues/239) [#346](https://github.com/zalando/dress-code/issues/346) [#347](https://github.com/zalando/dress-code/issues/347)
+
+
+### Features
+
+* **checkbox:** makes the checkboxes accessible ([f0990df](https://github.com/zalando/dress-code/commit/f0990df))
+* **grid:** Add utility classes to hide/show at different breakpoint sizes ([a5a875b](https://github.com/zalando/dress-code/commit/a5a875b)), closes [#287](https://github.com/zalando/dress-code/issues/287)
+* **grid:** Improve functionality and demo ([787c292](https://github.com/zalando/dress-code/commit/787c292))
+* **icons:** Add bar-chart, line-chart, pointer icons ([#356](https://github.com/zalando/dress-code/issues/356)) ([e863f1a](https://github.com/zalando/dress-code/commit/e863f1a))
+* **messages:** Update design of messages. ([2055e43](https://github.com/zalando/dress-code/commit/2055e43)), closes [#329](https://github.com/zalando/dress-code/issues/329)
+* **radio:** makes the radio button accessible ([b17b582](https://github.com/zalando/dress-code/commit/b17b582))
+* **tab:** add tab atom to the dress-code ([03f5711](https://github.com/zalando/dress-code/commit/03f5711))
+* **tab:** add tab atom to the dress-code ([0549bf5](https://github.com/zalando/dress-code/commit/0549bf5))
+* **tables:** Make table responsiveness optional ([#330](https://github.com/zalando/dress-code/issues/330)) ([ba80a3a](https://github.com/zalando/dress-code/commit/ba80a3a)), closes [#222](https://github.com/zalando/dress-code/issues/222)
+* **textarea:** add styles for error state ([d1cde40](https://github.com/zalando/dress-code/commit/d1cde40))
+
+
+### BREAKING CHANGES
+
+* breakpoints: * $dc-large-width breakpoint variable renamed to $dc-medium-width
```

Comment:
@dami-gg Can we fix the formatting here and make the breaking change really visible?